.ci.yaml

```diff
@@ -54,7 +54,6 @@ platform_properties:
       device_type: none
       cpu: x86
       os: Mac-12
-      xcode: 14e222b
       $flutter/osx_sdk : >-
         {
           "sdk_version": "14e222b"
```

Comment:
Do you need the runtime versions listed anywhere?

Reply:
No on the platform level. We have only one target needing runtimes. See `Mac Unopt` below.